a10_octavia/controller/worker/tasks/health_monitor_tasks.py

```diff
@@ -44,13 +44,15 @@ def execute(self, listeners, health_mon, vthunder):
         args = utils.meta(health_mon, 'hm', {})
 
         try:
+            pdata = CONF.a10_health_manager.post_data
```

Comment:
Bit of a nitpick, but it is cleaner here to just define it as `post_data` so long as it doesn't push us over line length limit.

We don't have to use the variable anywhere beyond this one section so there isn't a need to truncate

Reply:
updated